lib/jazzy/sourcekitten.rb

```diff
@@ -48,7 +48,8 @@ def autolink_block(doc_url, middle_regex, after_highlight)
 module Jazzy
   # This module interacts with the sourcekitten command-line executable
   module SourceKitten
-    @undocumented_abstract = Markdown.render('Undocumented').freeze
+    @undocumented_abstract = Markdown.render(Config.instance.undocumented_text)
```

Comment:
bcos you set the default to 'Undocumented' we can always use the config setting without any "if"s, this part can be like
```ruby
def self.undocumented_abstract
  @undocumented_abstract ||= Markdown.render(Config...)
end
```